src/coreclr/tools/aot/ILCompiler/ILCompiler_crossarch.csproj

```diff
@@ -0,0 +1,8 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <CrossHostArch>x64</CrossHostArch>
```

Comment:
Always x64? We don't have to worry about arm64?

Reply:
Changed to `$(BuildArchitecture)` to futureproof for the day when we crossbuild from a non-x64 host operating system. Also updated in the crossgen2 project.